src/main.rs

```diff
@@ -137,10 +137,8 @@ fn run_parallel(args: &Arc<Args>) -> Result<u64> {
             Continue
         })
     });
-    if !args.paths().is_empty() && paths_searched.load(Ordering::SeqCst) == 0 {
-        if !args.no_messages() {
-            eprint_nothing_searched();
-        }
+    if !args.paths().is_empty() && paths_searched.load(Ordering::SeqCst) == 0 && !args.no_messages() {
```

Comment:
Please drop this entire commit. I prefer the existing style.

Reply:
done :)